### ISSUE TITLE
Make Alertmanager configurable by this component 

### ISSUE THREAD

**Issue by corvus-ch** [appuio/component-openshift4-monitoring#3]
## Context

The [OpenShift 4 documentation](https://docs.openshift.com/container-platform/latest/monitoring/cluster_monitoring/configuring-the-monitoring-stack.html#configuring-alertmanager) explains how to configure Alertmanager. This is done by creating the secret ` alertmanager-main` within the namespace `openshift-monitoring`.

The key `alertmanager.yaml` within that secret should be configurable by this component.

**Comment by corvus-ch** [comment]
```yaml
global:
  resolve_timeout: 5m
route:
  group_wait: 30s
  group_interval: 5m
  repeat_interval: 12h
  receiver: default
  routes:
  - match:
      alertname: Watchdog
    repeat_interval: 5m
    receiver: watchdog
receivers:
- name: default
- name: watchdog
```

The above snipped is the default Alertmanager config taken from the OpenShift documentation. Should we add this as the default to this component? Alternatively, we could omit the config and do nothing.

**Comment by srueg** [comment]
Closed by #6
